CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - fixed ability to crash the game with extreme FOV values (regression from 0.9)
 - fixed double "Fly mode enabled" message when using `/fly` console command (regression from 4.0)
 - fixed crash in the `/set` console command (regression from 4.4)
+- fixed the main menu music being quiet or mute if a level was exited while underwater and the quiet, full but no ambient, quiet but no ambient, or none underwater music behavior option was set (regression from 4.4)
```

Comment:
One more comment – Can we shorten this a bit? For example `fixed main menu music volume when exiting underwater with certain music settings`, and link to the issue #1540.